contracts/access/AccessController.sol

```diff
@@ -3,15 +3,15 @@ pragma solidity 0.8.23;
 
 import { UUPSUpgradeable } from "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
 // solhint-disable-next-line max-line-length
-import { AccessManagedUpgradeable } from "@openzeppelin/contracts-upgradeable/access/manager/AccessManagedUpgradeable.sol";
 
 import { IAccessController } from "../interfaces/access/IAccessController.sol";
 import { IModuleRegistry } from "../interfaces/registries/IModuleRegistry.sol";
 import { IIPAccountRegistry } from "../interfaces/registries/IIPAccountRegistry.sol";
 import { IModuleRegistry } from "../interfaces/registries/IModuleRegistry.sol";
-import { IPAccountChecker } from "../lib/registries/IPAccountChecker.sol";
 import { IIPAccount } from "../interfaces/IIPAccount.sol";
+import { ProtocolPausableUpgradeable } from "../pause/ProtocolPausableUpgradeable.sol";
 import { AccessPermission } from "../lib/AccessPermission.sol";
+import { IPAccountChecker } from "../lib/registries/IPAccountChecker.sol";
```

Comment:
why move this line here?

Reply:
OCD? it sorts the import ../lib with ../lib and ../interfaces with ../interfaces